OasisDEX/oasis-borrow
# Loading all active triggers from cache instead of from blockchain

# [Replace implementation of automationTriggersData$ in app context](https://app.shortcut.com/oazo-apps/story/3203/replace-implementation-of-automationtriggersdata-in-app-context)

  
## Changes 👷‍♀️
  
- Replacing implementation with cache is in progress
  
## How to test 🧪
  
- Go to protection tab using goerli or alternatively hardhat if you have cache running separately using harhdat
- in network tab see responses from cache for used network with stoploss triggers queried from database using graphql
![image](https://user-images.githubusercontent.com/6871923/150854788-4d800627-fc8c-456f-b454-e918264ec0e0.png)
- Be able to set stoploss using cache instead of calling blockchain
- 
![image](https://user-images.githubusercontent.com/6871923/150855177-4f6ec3ee-9b03-45e4-9bf1-18f1fd622372.png)

- Be able to cancel stop loss, so there's no regression
- 
![image](https://user-images.githubusercontent.com/6871923/150855556-037e6a52-3d8b-4223-ba83-32850ebd3cdd.png)




    
## Definition of done ✔️
